models/staging/stg_dbt_models.sql

```diff
@@ -37,17 +37,7 @@ surrogate_key as (
 
     select
         {{ dbt_utils.surrogate_key(['command_invocation_id', 'node_id']) }} as manifest_model_id,
-        command_invocation_id,
-        artifact_generated_at,
-        node_id,
-        model_database,
-        model_schema,
-        name,
-        depends_on_nodes,
-        package_name,
-        model_path,
-        checksum,
-        model_materialization
+        flatten.*
```

Comment:
i dont think we need to specify `flatten.*` since we arent joining to another table. this can just be `*`